scripts/docker-all.sh

```
@@ -0,0 +1,211 @@
+  command -v "$1" >/dev/null 2>&1
+}
+
+# Function to wait for a service to be healthy
+wait_for_container() {
+  local container=$1
+  local max_attempts=${2:-60}
+  local attempt=1
+
+  echo -e "${YELLOW}Waiting for $container to be healthy...${NC}"
+  while [ $attempt -le $max_attempts ]; do
+    local status=$(docker inspect --format='{{.State.Health.Status}}' "$container" 2>/dev/null || echo "not_found")
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)